README.md

```diff
@@ -1,3 +1,226 @@
-TuxedoJS
-========
+# TuxedoJS
+
 ![Tux Logo](Tux-full-logo.png "Tux-Logo")
+
+
+> A Front-End Javascript Framework built on Facebook’s powerful React view layer and Flux architecture.
+
+React and Flux are two novel and exciting approaches to front-end development. TuxedoJS implements a more accessible and semantic toolkit that leverages these technologies, abstracting away complexity and extending core functionality. TuxedoJS comes from Tux - a merging of React and Flux.
+
+## Table of Contents
+
+1. [Usage](#usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+    1. [Installing Dependencies](#installing-dependencies)
+    1. [Getting Started](#getting-started)
+    1. [Tasks](#tasks)
+    1. [Roadmap](#roadmap)
+1. [Graceful Degradation](#graceful-degradation)
+1. [Credits and Tech Stack](#credits-and-tech-stack)
+1. [Contributing](#interested-in-contributing)
+1. [Team](#team)
+
+## Usage
+
+TuxedoJS is a modular framework that provides tools for easily rendering components, client-side routing, animating transitions, validating inputs, and managing app data.  Install it via a package manager and include the modules you need to quickly start building your app.
+
+See the full doc site [here](http://www.tuxedojs.org).
+
+## Requirements
+
+- Node 0.10.x
+
+## Development
+
+### Installing Dependencies
+
+From within the root directory:
+
+    npm install
+
+### Getting Started
+
+To get started hacking with Tux, lets see what some parts of a to-do component might look like inside a To-Do Application built with Tux.
```

Comment:
This is a bit confusing wording.  Try changing to: Let's try building a simple ToDo application in Tux.